lib/resty/upstream/healthcheck.lua

```diff
@@ -234,47 +235,39 @@ local function check_peer(ctx, id, peer, is_backup)
         ok, err = sock:sslhandshake(nil, ctx.host, ctx.ssl_verify)
         if not ok then
             sock:close()
-            return peer_error(ctx, is_backup, id, peer,
-                              "failed to ssl handshake to ", name, ": ", err)
+            return peer_error(ctx, is_backup, id, peer, "failed to ssl handshake to ", name, ": ", err)
```

Comment:
Changing the code style adds to the reviewer's workload

Reply:
how do you format the code？

Reply:
I use lua-format. I added the `.lua-format` file for that reason.
You can not have a line as long as yours is AND have a line length of 80. also you can not expect me or any other contributor to know, what to keep and what to format. Provide me a ruleset for formatting and we can translate it into a lua-format-config. If you set the line length to 80, without stating it in the repository, how am I supposed to know?

Reply:
I would expect to have an easy way of (auto-)formatting my code. So let‘s not call it 'changing the code style' when all I am doing is trying to get this code formatted in a repeatable way. The lua-format file can help with that. So let’s work together and find a way of implementing this.
This would maybe change some lines dramatically once, but then all the future PRs can use the same formatter and won’t change the adapted lines automatically again. So currently your work is much more on every PR, because you don’t provide me a way of what to use for formatting my code. And I don’t want to always undo the formatting manually on lines that don’t follow any code style ruleset.